setup.py

```diff
@@ -15,8 +16,11 @@
     long_description_content_type='text/markdown',
     py_modules=['ordered_set'],
     package_data={'': ['MIT-LICENSE']},
+    data_files=(('lib/python{}.{}/site-packages'.format(*sys.version_info[:2]), ['ordered_set.pyi']),
```

Comment:
This doesn't look right to me.

I tried searching for things related to this. I found https://github.com/python/typing/issues/84#issuecomment-317217346, where lines like this appear to be a hack that MyPy developers were using to experiment with installable types.

The "ideal UX for library authors" suggests that this should be unnecessary, that .pyi files should just be pip installed the same way as .py files. If this doesn't work yet, then I think that is something for pip and mypy to work out between them, instead of every package maintainer adding fragile-looking path acrobatics to their library. On the other hand, if this really is just boilerplate that the Python devs promise will keep working, just point me to the guide.